src/interactive/azext_interactive/azclishell/layout.py

```diff
@@ -76,11 +78,15 @@ def get_prompt_tokens(self, _):
             prompt = 'az {}>> '.format(self.shell_ctx.default_command)
         else:
             prompt = 'az>> '
-        return [(Token.Az, prompt)]
+        return [(Token.Az, self.prompt_prefix + prompt)]
 
     def create_tutorial_layout(self):
         """ layout for example tutorial """
-        lexer, _, _ = get_lexers(self.shell_ctx.lexer, None, None)
+        lexer, _, _, _ = get_lexers(self.shell_ctx.lexer, None, None, None)
+
+        if not any(isinstance(processor, DefaultPrompt) for processor in self.input_processors):
+            self.input_processors.append(DefaultPrompt(self.get_prompt_tokens))
```

Comment:
Could you please add a comment to explain why do we need this logic?

Reply:
Done